test.properties

```diff
@@ -3763,6 +3772,33 @@
             (testray.main.component.name ~ "User Interface")\
         )
 
+    test.batch.run.property.query[functional-upgrade-tomcat90-db2111-jdk8][frontend]=\
+        (database.types ~ db2) AND \
+        (testray.main.component.name == "Upgrades Remote Apps")
+
+    test.batch.run.property.query[functional-upgrade-tomcat90-mariadb104-jdk8][frontend]=\
+        (database.types ~ mariadb) AND \
+        (testray.main.component.name == "Upgrades Remote Apps")
+
+    test.batch.run.property.query[functional-upgrade-tomcat90-mysql57-jdk8][frontend]=\
+        (testray.main.component.name == "Upgrades Remote Apps")
+
+    test.batch.run.property.query[functional-upgrade-tomcat90-oracle193-jdk8][frontend]=\
+        (database.types ~ oracle) AND \
+        (testray.main.component.name == "Upgrades Remote Apps")
+
+    test.batch.run.property.query[functional-upgrade-tomcat90-postgresql134-jdk8][frontend]=\
+        (database.types ~ postgresql) AND \
+        (testray.main.component.name == "Upgrades Remote Apps")
+
+    test.batch.run.property.query[functional-upgrade-tomcat90-sqlserver2017-jdk8][frontend]=\
+        (database.types ~ sqlserver) AND \
+        (testray.main.component.name == "Upgrades Remote Apps")
+
+    test.batch.run.property.query[functional-upgrade-tomcat90-sybase160-jdk8][frontend]=\
+        (database.types ~ sybase) AND \
+        (testray.main.component.name == "Upgrades Remote Apps")
+
```

Comment:
I was wondering if our PQL is smart enough and this section could be simplified to something like the following and do the same thing:
```
test.batch.run.property.query[functional-upgrade-*][frontend]=\
        (testray.main.component.name == "Upgrades Remote Apps")
```

Reply:
ohhh, I can test it out